subsys/bluetooth/host/Kconfig

```diff
@@ -1056,13 +1056,24 @@ config BT_RFCOMM_L2CAP_MTU
 
 config BT_RFCOMM_TX_MAX
 	int "Maximum number of pending TX buffers for RFCOMM"
-	default BT_MAX_CONN
-	range BT_MAX_CONN 255
+	default BT_CONN_TX_MAX
+	range BT_CONN_TX_MAX 255
 	help
 	  Maximum number of pending TX buffers that have an associated
 	  sending buf. Normally this can be left to the default value, which
 	  is equal to the number of session in the stack-internal pool.
 
+config BT_RFCOMM_DLC_STACK_SIZE
```

Comment:
I just realized that we're missing `subsys/bluetooth/host/classic/Kconfig` all of the Classic Kconfig options should be moved there instead of adding new ones to `host/Kconfig`

Reply:
I added Kconfig.classic for Bluetooth classic Kconfig